doc/string/rpartition.rdoc

```diff
@@ -0,0 +1,22 @@
+Returns a 3-element array of substrings of +self+.
+
+Matches a pattern against +self+; the pattern is:
+
+- +string_or_regexp+ itself, if it is a Regexp.
+- <tt>Regexp.quote(string_or_regexp)</tt>, if +string_or_regexp+ is a string.
+
+If the pattern is matched, returns pre-match, last-match, post-match:
+
+  'hello'.rpartition('l')      # => ["hel", "l", "o"]
+  'hello'.rpartition('ll')     # => ["he", "ll", "o"]
+  'hello'.rpartition('h')      # => ["", "h", "ello"]
+  'hello'.rpartition('o')      # => ["hell", "o", ""]
+  'hello'.rpartition('')       # => ["hello", "", ""]
+  'тест'.rpartition('т')       # => ["тес", "т", ""]
+  'こんにちは'.rpartition('に')  # => ["こん", "に", "ちは"]
+
```

Comment:
I think it's valuable to have the following comment from the original description:

>  The match from the end means starting at the possible last position, not the last of longest matches.

And the following example:

```ruby
'hello'.rpartition(/l+/) #=> ["hel", "l", "o"]
```